components/climber.py

```diff
@@ -6,6 +6,8 @@
 
 class ClimberComponent:
     GEAR_RATIO = 1 / 48  # using a Neo with 4:1 4:1 3:1 ratio
+    MOTOR_ROT_TOP_LIMIT = 7.110515
+    MOTOR_ROT_BOTTOM_LIMIT = 0
```

Comment:
These are not in motor rotations. We also typically use rev for revolutions rather than rot